gluten-core/src/main/scala/org/apache/gluten/expression/UnaryExpressionTransformer.scala

```diff
@@ -203,43 +102,32 @@ case class RandTransformer(
     substraitExprName: String,
     explicitSeed: ExpressionTransformer,
     original: Rand)
-  extends ExpressionTransformerWithOrigin {
+  extends LeafExpressionTransformer {
```

Comment:
This is an another example to show how to work with `ExpressionTransformer`, rand is a unary expression in vanilla Spark but native does not accpet arg, then we mark it as leaf expression transformer.